pydantic_extra_types/coordinate.py

```diff
@@ -0,0 +1,202 @@
+from typing import Any, Callable, ClassVar, Tuple, Type, Union
+
+from pydantic import GetCoreSchemaHandler, GetJsonSchemaHandler
+from pydantic._internal import _repr
+from pydantic.json_schema import JsonSchemaValue
+from pydantic_core import CoreSchema, PydanticCustomError, core_schema
+
+LatitudeType = Union[str, int, float]
+
+
+class Latitude(float):
+    ge: ClassVar[float] = -90.00
+    le: ClassVar[float] = 90.00
```

Comment:
I'd rather used `min` and `max` instead of `ge`/`le`. IMO, it express the meaning of attributes better.

Reply:
Done